crates/sui-json-rpc-types/src/sui_transaction.rs

```diff
@@ -1619,19 +1619,33 @@ impl From<Command> for SuiCommand {
     }
 }
 
-/// An argument to a transaction in a programmable transaction block
+/// An argument to a transaction in a programmable transaction block.
 #[derive(Debug, Copy, Clone, Serialize, Deserialize, JsonSchema, PartialEq, Eq)]
 pub enum SuiArgument {
     /// The gas coin. The gas coin can only be used by-ref, except for with
     /// `TransferObjects`, which can use it by-value.
     GasCoin,
-    /// One of the input objects or primitive values (from
-    /// `ProgrammableTransactionBlock` inputs)
+    /// One of the input objects or primitive values (from `ProgrammableTransactionBlock`
+    /// inputs).
+    ///
+    /// Indexing is zero-based, meaning that `Input(0u16)` refers to the input of the PTB's first
+    /// command, `Input(1u16)` to the second, and so on.
```

Comment:
Inputs are shared among all commands/transactions in the PTB.

```suggestion
    /// Indexing is zero-based, meaning that `Input(0u16)` refers to the first PTB input,
    /// `Input(1u16)` to the second, and so on.
```